docs/src/references/c8y-configuration-management.md

```diff
@@ -93,49 +98,56 @@ by the configuration of thin-edge:
 * `tedge config get mqtt.port`: the TCP port of the local MQTT bus.
 * `tedge config get tmp.path`: the directory where the files are updated
   before being copied atomically to their targets.
-* `tedge config get log.path`: the directory used for logging.
 
 ## Usage
 
 ```shell
-$ c8y_configuration_plugin
+$ c8y_configuration_plugin --help
+c8y_configuration_plugin 0.6.2
 Thin-edge device configuration management for Cumulocity
 
 USAGE:
     c8y_configuration_plugin [OPTIONS]
 
 OPTIONS:
         --config-dir <CONFIG_DIR>      [default: /etc/tedge]
-        --config-file <CONFIG_FILE>    [default: /etc/tedge/c8y/c8y_configuration_plugin.toml]
+        --config-file <CONFIG_FILE>    [default: $CONFIG_DIR/c8y/c8y-configuration-plugin.toml]
```

Comment:
We don't even need this `--config-file` option, when the `--config-dir` option is available, right? The config file will always be created under `$CONFIG_DIR/c8y/c8y-configuration-plugin.toml` irrespective of whether the `--config-dir` is specified or not. If `CONFIG_DIR` is not specified, its default value would be `/etc/tedge`.

Reply:
I agree that is a bit more complicated than strictly required. But yes you can specify a `--config-file` that is not inside the `--config-dir`.